server/svix-server/src/worker.rs

```diff
@@ -326,6 +330,33 @@ pub async fn worker_loop(
     }
 }
 
+/// Nullifies the payload column for expired messages
+async fn clean_expired_messages(
```

Comment:
I wouldn't put this in this file. I agree the name worker.rs is slightly confusing, but the intention is just for the webhook workers.

It's large enough already, and this worker is completely unrelated.